.drone.yml

```diff
@@ -33,9 +33,55 @@ steps:
   - mvn sonar:sonar -Dsonar.host.url=$${SONAR_HOST} -Dsonar.login=$${SONAR_TOKEN} -DprojectKeySuffix=$${projectSuffix}
   when:
     event:
-      exclude:
+      exlcude:
+      - pull_request
+
+- name: publish
+  image: maven:3.8-openjdk-17
+#  depends_on:
+#    - sonar
+  environment:
+    ARTIFACTORY_TOKEN:
+        from_secret: artifactory_token
+  commands:
+  - mvn -s ./jparest_settings.xml -f pom.xml install -B org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy -DskipTests
+  when:
+    branch: main
+#    event:
+#      exlcude:
+#      - pull_request
+
+- name: Git Tag
+  image: jlrigau/maven-git
```

Comment:
This project doesn’t look to have been recently maintained 🤔 How confident are we that we should use it? Is there a fresher alternative?